src/omv/boards/OPENMV_RT1060/omv_boardconfig.h

```diff
@@ -211,4 +212,11 @@
 #define OMV_CSI_POWER_PIN               (&omv_pin_DCMI_POWER)
 #define OMV_CSI_FSYNC_PIN               (&omv_pin_DCMI_FSYNC)
 
+#define OMV_CSI_HAS_BSWAP               (1)
+
+#define OMV_CSI_DMA                     (DMA0)
+#define OMV_CSI_DMA_MUX                 (DMAMUX)
+#define OMV_CSI_DMA_CHANNEL_START       (4U)
+#define OMV_CSI_DMA_CHANNEL_COUNT       (2U)
```

Comment:
If you'll keep those options please move them after `OMV_CSI_BASE`

Reply:
Done